build.gradle

```diff
@@ -846,6 +855,81 @@ project(':core') {
     duplicatesStrategy 'exclude'
   }
 
+  apply plugin: 'nebula.rpm'
+
+  // Mostly the same as the releaseTarGz, but with different prefixing, links, and without the configs
+  tasks.create(name: 'releaseRpm', type: Rpm) {
+    os LINUX
+    type BINARY
+    release "$rpmRelease"
+    license 'Apache License 2.0'
+
+    user "kafka"
+    group "kafka"
+
+    from(project.file("$rootDir/bin")) { into "/usr/share/kafka/bin/" }
```

Comment:
Should we make a separate task which has common copy commands from both releaseTar and releaseRPM and just use depends on that common task. 

Basically push the common functionality to the upstream . This will help in code maintainability no ?